content/tutorials/manage-account-settings/use-tickets.md

```diff
@@ -0,0 +1,372 @@
+
+You use the `ledger` event type in RippleAPI to trigger your code to run whenever there is a new validated ledger version. For example:
+
+```js
+// signed.id is the hash we're waiting for
+// min_ledger is the validated ledger index at time of first submission
+// max_ledger is the transaction's LastLedgerSequence value
+let tx_status = ""
+api.on('ledger', async (ledger) => {
+  console.log("Ledger version", ledger.ledgerVersion, "was validated.")
+  if (!tx_status) {
+    try {
+      tx_result = await api.request("tx", {
+          "transaction": signed.id,
+          "min_ledger": min_ledger,
+          "max_ledger": max_ledger
+      })
+
+      if (tx_result.validated) {
+        console.log("Got validated result:", tx_result.meta.TransactionResult)
+        tx_status = "validated"
+      } else {
+        // Transaction found, but not yet validated. No change.
+      }
+    } catch(e) {
+      if (e.data.error == "txnNotFound") {
+        if (e.data.searched_all) {
+          console.log(`Tx not found in ledgers ${min_ledger}-${max_ledger}`)
+          tx_status = "rejected"
+          // This result is final if min_ledger and max_ledger are correct
+        } else {
+          if (max_ledger > ledger.ledgerVersion) {
+            // Transaction may yet be confirmed. Keep waiting.
+          } else {
+            console.log("Can't get final result. Check a full history server.")
+            tx_result = "unknown - check full history"
+          }
+        }
+      } else {
+        // Unknown error; pass it back up
+        throw e
+      }
+    }
+  }
+})
+```
+
+{{ start_step("Wait") }}
+<table>
+  <tr>
+    <th>Transaction Hash:</th>
+    <td id="waiting-for-tx"></td>
+  </tr>
+  <tr>
+    <th>Latest Validated Ledger Version:</th>
+    <td id="current-ledger-version">(Not connected)</td>
+  </tr>
+  <tr>
+    <th>Ledger Version at Time of Submission:</th>
+    <td id="earliest-ledger-version">(Not submitted)</td>
+  </tr>
+  <tr>
+    <th><code>LastLedgerSequence</code>:</th>
+    <td id="lastledgersequence">(Not prepared)</td>
+  </tr>
+  <tr id="tx-validation-status">
+  </tr>
+</table>
+{{ end_step() }}
+
+
+### (Optional) Intermission
+
+The power of Tickets is that you can send other transactions during this time, and generally carry on with your account's normal business during this time. If you are planning on using a given Ticket, then as long as you don't use that Ticket for something else, you're free to continue using your account as normal. When you want to send the transaction using a Ticket, you can do that in parallel with other transactions using regular sequence numbers or different Tickets, and submit any of them in any order when you're ready.
```

Comment:
> The power of Tickets is that you can send other transactions **during this time**, and generally carry on with your account's normal business **during this time**.

We could stand to lose one "during this time" in the opening sentence. 

The whole paragraph could probably be tightened up a little more, though I know that's easier suggested than done :)